<?xml version="1.0"?>
<launch>

    <include file="$(find exp_assignment3)/launch/gmapping.launch"/>
    <include file="$(find exp_assignment3)/launch/move_base.launch"/>


    <node pkg="armor" type="execute" name="armor_service" args="it.emarolab.armor.ARMORMainService"/> 
    <node name="final_oracle" pkg="exp_assignment3" type="final_oracle" />
    <node name="move_arm" pkg="exp_assignment3" type="move_arm" />
    <!-- <node name="marker_publisher" pkg="aruco_ros" type="marker_publisher"/> -->
    <!-- <node name="ontology_settings" pkg="exp_assignment3" type="OntologySettings.py"/>
    <node name="cluedo_fsm" pkg="exp_assignment3" type="cluedo_fsm.py" /> -->
    <node name="smach_viewer" pkg="smach_viewer" type="smach_viewer.py"/>
    <!-- <node name="user_interface" pkg="exp_assignment3" type="user_interface.py" output="screen"/> -->

    
            
</launch>